src/components/Cell/Cell.tsx

```diff
@@ -37,14 +37,19 @@ const Cell = ({id, toggleModal}: GridCell) => {
         }
     }
 
+    const hoverStyle = isOver && !isDisabled && {
```

Comment:
love this hover effect, however, it's a bit difficult to recognize due to the size of the draggable objects. perhaps we could find a way to shrink draggable objects to match the size of the cells?

Reply:
I agree and there may be a way, but it may take a ton of research to figure out we don't have time for.  I'll make a ticket tho!